Source/JavaScriptCore/b3/testb3.h

```diff
@@ -117,7 +120,7 @@ extern Lock crashLock;
     
 #define CHECK_EQ(x, y) do { \
     auto __x = (x); \
-    auto __y = (y); \
+    decltype(__x) __y = (y); \
```

Comment:
Why do we need this change?

Reply:
This avoids having to add some casts for literals. As far as I can tell, it shouldn't implicitly convert anything, it just avoids having to add some extra casts to literals.

Reply:
I fixed this